00-open-a-shapefile.Rmd

```diff
@@ -159,31 +155,26 @@ aoiBoundary <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
 ```
 
 After loading a new data set we also want to view the attributes of the layer.
-This can be done simple by entering the file name.
+This can be done simple by entering the file name. Alternatively, we can use the `attributes()` function.
 
 ``` {r Shapefile-attributes}
 #view attributes of the layer
 aoiBoundary
 ```
 
-Here we see that R interprets the data as a SpatialPolygonsDataFrame data class
-and information about the coordinate reference system including that the 
-coordinates  are UTMs from zone 18.  This helps us interpret the previous line 
-giving the extent of the file.  We will discuss metadata and attributes in more
-detail a bit later. Finally we see there is one variable named "id" that appears
-to consists of min and max values of 1.
+Here `R` prints a number of attributes including information about the object's coordinate reference system. Note that the objects coordinates are UTM projected from zone 18. This information is critical to interpreting the object extent printed on the previous line. Also, note that our object only contains a single feature. We will discuss metadata and attributes in more
```

Comment:
Edits:

Here `R` prints a number of attributes including information about the object's `coordinate reference system (CRS)`. Note that the object coordinates are in the projection: UTM zone 18N.